src/main/scala/cromwell/engine/db/slick/JesJobComponent.scala

```diff
@@ -32,14 +32,20 @@ trait JesJobComponent {
       executionId, unique = true)
   }
 
-  val jesJobs = TableQuery[JesJobs]
+  protected val jesJobs = TableQuery[JesJobs]
 
   val jesJobsAutoInc = jesJobs returning jesJobs.
     map(_.jesJobId) into ((a, id) => a.copy(jesJobId = Some(id)))
 
-  val jesJobByID = Compiled(
-    (id: Rep[Int]) => for {
+  val jesJobsByExecutionId = Compiled(
+    (executionId: Rep[Int]) => for {
       jesJob <- jesJobs
-      if jesJob.jesJobId === id
+      if jesJob.executionId === executionId
     } yield jesJob)
+
+  val jesJobJesJobIdsAndJesStatusesByExecutionId = Compiled(
```

Comment:
Changed.

FYI for posterity: I'm terrible at naming. Was trying to use stay consistent, similar in naming to Grails' [dynamic finders](http://grails.github.io/grails-doc/3.0.0/ref/Domain%20Classes/findAllBy.html).